### ISSUE TITLE
ImportError: /lib64/libc.so.6: version `GLIBC_2.32' not found

### ISSUE THREAD

**Issue by devpramod** [intel-analytics/BigDL#10517]
Following instructions from [https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/LLM-Finetuning/QLoRA/simple-example](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/LLM-Finetuning/QLoRA/simple-example) to set up exaple

Facing the following error:

`import linear_q4_0
ImportError: /lib64/libc.so.6: version `GLIBC_2.32' not found`

**Comment by Fred-cell** [comment]
upgrade the GLIC version

**Comment by qiyuangong** [comment]
> GLIBC_2.32' not found`

Hi @devpramod 
This error is caused by glibc out of date. In most cases, it's due to OS out of date.

**But don't upgrade glibc because it's OS and kernel-related. It may cause your system.**

Please try docker container  **https://github.com/intel-analytics/BigDL/tree/main/docker/llm/finetune/qlora/xpu/docker**. Or install glibc in conda.

**Comment by devpramod** [comment]
Thanks @qiyuangong 
Is it possible to install glibc through conda only?

**Comment by qiyuangong** [comment]
> Thanks @qiyuangong Is it possible to install glibc through conda only?

Yes. It's possible but still risky (it may also crash your conda env or application). 
Please use the docker solution. It's much safer.

You can use this command to check your glibc version.
```bash
strings /lib64/libc.so.6 | grep GLIBC
```
